src/main/java/org/apache/accumulo/testing/continuous/ContinuousIngest.java

```diff
@@ -173,6 +181,18 @@ protected static void doIngest(AccumuloClient client, long rowMin, long rowMax,
     log.info("DELETES will occur with a probability of {}",
         String.format("%.02f", deleteProbability));
 
+    zipfianEnabled = Boolean.parseBoolean(testProps.getProperty("test.ci.ingest.zipfian.enabled"));
+
+    if (zipfianEnabled) {
+      minSize = Integer.parseInt(testProps.getProperty("test.ci.ingest.zipfian.min.size"));
+      maxSize = Integer.parseInt(testProps.getProperty("test.ci.ingest.zipfian.max.size"));
```

Comment:
Should these be `Long.parseLong()` instead or is Integer enough?

Reply:
I think Integer is enough. The place where this is used (`rnd.nextZipf()`) only accepts an int anyways:
https://github.com/apache/accumulo-testing/blob/db3835e37f9ebcd336c9cc62578366539f15aa24/src/main/java/org/apache/accumulo/testing/continuous/ContinuousIngest.java#L349-L350